kitsune/dashboards/jinja2/dashboards/contributors.html

```diff
@@ -64,7 +64,11 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
     {% include 'dashboards/includes/kb_overview.html' %}
   </table>
   <div class="table-footer">
-    <a href="{{ url('dashboards.contributors_overview')|urlparams(product=product.slug, category=category) }}">{{ _('Complete overview...') }}</a>
+    {% set pslug=product.slug %}
+    {% if product==None %}
+      {% set pslug=None %}
+    {% endif %}
```

Comment:
We can reduce this to one line with a `{ set a = b if c else e }` statement.

Reply:
correct, done changes.